jcl/src/java.base/share/classes/java/lang/invoke/BruteArgumentMoverHandle.java

```diff
@@ -137,11 +137,11 @@ void compareWithArgumentMover(ArgumentMoverHandle left, Comparator c) {
 	final Object[] extra;
 
 	// Save a level of indirection for the first few objects.
-	Object   extra_L0;
-	Object   extra_L1;
-	Object   extra_L2;
-	Object   extra_L3;
-	Object   extra_L4;
+	final Object   extra_L0;
+	final Object   extra_L1;
+	final Object   extra_L2;
+	final Object   extra_L3;
+	final Object   extra_L4;
```

Comment:
Does the comment just below apply to this as well?
https://github.com/eclipse/openj9/blob/5f3ace63af97087053a9dc40e62cf10931bc885f/jcl/src/java.base/share/classes/java/lang/invoke/BruteArgumentMoverHandle.java#L146-L148

What does marking these object fields as `final` do to the sharedThunks?

Reply:
It won't affect the sharedThunks. These fields won't be folded in sharedThunks because the receiver handle is not known object in sharedThunks.